save-backend/src/main/kotlin/com/saveourtool/save/backend/security/OrganizationPermissionEvaluator.kt

```diff
@@ -78,7 +78,7 @@ class OrganizationPermissionEvaluator {
     ): Boolean {
         val selfRole = lnkUserOrganizationService.getGlobalRoleOrOrganizationRole(authentication, organization)
         val otherRole = lnkUserOrganizationService.findRoleByUserIdAndOrganization(otherUser.id!!, organization)
-        return isOrganizationAdminOrHigher(selfRole) && hasAnotherUserLessPermissions(selfRole, otherRole) &&
+        return selfRole.isHigherOrEqualThan(Role.OWNER) || isOrganizationAdminOrHigher(selfRole) && hasAnotherUserLessPermissions(selfRole, otherRole) &&
```

Comment:
`isOrganizationAdminOrHigher(selfRole)` can be refactored to `selfRole.isHigherOrEqualThan(Role.ADMIN)` and method can be removed (I guess)

Reply:
Definitely